custom/conf/app.example.ini

```diff
@@ -2559,6 +2559,9 @@ LEVEL = Info
 ;PROXY_URL =
 ;; Comma separated list of host names requiring proxy. Glob patterns (*) are accepted; use ** to match all hosts.
 ;PROXY_HOSTS =
+;; Enable the smtp proxy through when the proxy type is https://, socks://, all notifications through smtp emails
+;; will be affected. SMTP_PROXY_ENABLED takes effect with the precondition PROXY_ENABLED = true.
+;SMTP_PROXY_ENABLED = false
```

Comment:
Also sync this into docs/content/administration/config-cheat-sheet.en-us.md

Reply:
updated.

Reply:
thanks